app/dashboard/posts/create/page.tsx

```diff
@@ -19,15 +19,7 @@ import {
 import { store } from './action/store';
 
 export default async function Page() {
-  const { data: userData, error: userNotAuthenticated } =
-    await verify.loggedUser();
-
-  if (userNotAuthenticated) {
-    return redirect(
-      '/auth/signin?redirect_reason=not-authenticated',
-      RedirectType.replace,
-    );
-  }
```

Comment:
I removed this verification because it is already being done in `/dashboard/posts/layout.tsx`: https://github.com/SouzaGabriel26/onde-ir/blob/24977674201c457a18597dfd395f57743db11cbb/app/dashboard/posts/layout.tsx#L16-L21